node/node.d.ts

```diff
@@ -607,6 +607,7 @@ declare module "http" {
         removeHeader(name: string): void;
         write(chunk: any, encoding?: string): any;
         addTrailers(headers: any): void;
+        finished: boolean;
```

Comment:
@TonyPythoneer  I noticed you added stuff to this file . process.exitCode isn't in that file. Could you add it?